tests/xharness/Harness.cs

```diff
@@ -111,13 +111,39 @@ public class Harness
 		public string PeriodicCommandArguments { get; set; }
 		public TimeSpan PeriodicCommandInterval { get; set; }
 		// whether tests that require access to system resources (system contacts, photo library, etc) should be executed or not
-		public bool IncludeSystemPermissionTests { get; set; } = true;
+		public bool? IncludeSystemPermissionTests { get; set; }
```

Comment:
Is the bool?/nullable pattern something we commonly use for values that can have 3 states?

Reply:
It doesn't always make sense to use a nullable (sometimes an enum can be a better choice, in particular if the three states are not common, since the enum allows for a better description of each state), but we use it quite often.

This is one scenario where we use it a lot (option not set, option set to true, option set to false).